copilot-chat.el

```diff
@@ -539,6 +579,7 @@ This can be overrided by frontend."
 (defun copilot-chat-reset()
   "Reset copilot chat session."
   (interactive)
+  (copilot-chat-list-clear-buffers)
```

Comment:
during using, feel like it is more comfortable if it reset everything, including buffer list. (eg. switching to work on a different project). Up to discussion.

Reply:
buffer list is already cleared by copilot-chat--create. Did you find any problem without this line ?